tracpro/polls/views.py

```diff
@@ -376,7 +376,7 @@ def get_context_data(self, **kwargs):
 
             # Calculate all no-group or no-region activity
```

Comment:
Just noticing that this whole little section, as it already existed, seems fishy.  We would never have a contact with no groups, because we only sync contacts that have at least one group in our selected cohorts and one group in our selected panels.

This isn't the time to go tearing into other parts of the code, but maybe we should open a ticket to look into this at some point?  It doesn't *seem* like it could be doing anything useful.

Reply:
+1